opal/mca/if/linux_ipv6/configure.m4

```diff
@@ -28,7 +29,7 @@ AC_DEFUN([MCA_opal_if_linux_ipv6_CONFIG], [
     AC_MSG_CHECKING([if we are on Linux with TCP])
     # If we have struct sockaddr and we're on Linux, then we're
     # happy.
-    AS_IF([test "$opal_found_sockaddr" = "yes" && test "$opal_found_linux" = "yes"],
+    AS_IF([test "$opal_found_sockaddr_in" = "yes" && test "$oac_found_linux" = "yes"],
```

Comment:
Same question as above why do we care if IPV4 struct exists on an IPV6 module ?

Reply:
Same answer as above.